automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeGceSpec.scala

```diff
@@ -114,15 +115,14 @@ class RuntimeGceSpec
         implicit0(authToken: AuthToken) <- Ron.authToken()
         _ <- IO(withWebDriver { implicit driver =>
           withNewNotebook(clusterCopy, Python3) { notebookPage =>
+            notebookPage.executeCell("import tensorflow as tf")
```

Comment:
for some reason the test was previously trying to evaluate if the gpu string was in the output from this import step (instead of where we actually list the devices), so I pulled this step out

Reply:
Oh whoa this is the weirdest race condition, good find!